src/css/_utilities.scss

```diff
@@ -94,11 +96,15 @@
   order: $value;
```

Comment:
Nope, I noticed that. It's standard, though: https://drafts.csswg.org/css-flexbox-1/#order-property